packages/docusaurus-theme-search-algolia/src/theme/SearchBar/algolia.css

```diff
@@ -234,6 +234,10 @@
   }
 }
 
+.algolia-autocomplete pre {
```

Comment:
Algolia adds this invisible pre element, and our built-in styles add padding due to which extra spaces appears on the mobiles on the right (due to sticky positioned navabr)